src/uct/tcp/tcp_sockcm.c

```diff
@@ -25,6 +25,13 @@ ucs_config_field_t uct_tcp_sockcm_config_table[] = {
   {NULL}
 };
 
+void uct_tcp_sockcm_close_fd(int fd)
```

Comment:
good idea to have fatal for the error from `close()`
what about moving this code to `ucs/sys/sock.[ch]` and combine with the following in TCP?
https://github.com/openucx/ucx/blob/ed6b365dc9193dac8a4243f8c00b0ab8db368d89/src/uct/tcp/tcp_ep.c#L125